ci-operator/step-registry/openshift/e2e/alibabacloud/serial/openshift-e2e-alibabacloud-serial-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: openshift-e2e-alibabacloud-serial
+  steps:
+    pre:
+    - chain: ipi-alibabacloud-pre
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-alibabacloud-post
```

Comment:
I wonder why the serial workflows do not do the gather-core-dump chain. I don't see it in any of the other serial workflows that I looked at.